### ISSUE TITLE
ZLIB not found causing Travis and Install Failures

### ISSUE THREAD

**Issue by rbharath** [deepchem/deepchem#792]
Travis Tests are failing with ZLIB not found error (H/T @proteneer).

```
/home/travis/build/deepchem/deepchem/deepchem/data/datasets.py", line 11, in <module>
    from deepchem.utils.save import save_to_disk
  File "/home/travis/build/deepchem/deepchem/deepchem/utils/__init__.py", line 14, in <module>
    from rdkit import Chem
  File "/home/travis/anaconda/envs/deepchem/lib/python3.5/site-packages/rdkit/Chem/__init__.py", line 28, in <module>
    from rdkit.Chem.rdmolops import *
ImportError: /home/travis/anaconda/envs/deepchem/lib/python3.5/lib-dynload/../../libz.so.1: version `ZLIB_1.2.9' not found (required by /home/travis/anaconda/envs/deepchem/lib/python3.5/site-packages/rdkit/Chem/../../../.././libpng16.so.16)
```

@jfarlow reports on gitter that `nosetests` are failing on a fresh install with ```ImportError: .../libz.so version `ZLIB_1.2.0' not found, required by ..../libpng16.so.16``` so it seems like this is a general problem.

Is this a general error caused by an upstream update?

CC @lilleswing @peastman

**Comment by peastman** [comment]
Based on a web search, it sounds like this is a common problem.  It happens when one package installs its own version of zlib, and that causes other libraries to try to link to that version instead of the system one.  At least, that's one thing that can cause it.

Have any of our dependencies changed recently?  Or has travis updated the system version?

**Comment by lilleswing** [comment]
![selection_093](https://user-images.githubusercontent.com/1225605/29715331-eea796d2-8974-11e7-8924-7ab7788e6c90.png)

It doesn't look like any of our deps have changed since 08/08.  I went ahead and pinned all of our non-pinned deps here.

https://github.com/deepchem/deepchem/compare/master...lilleswing:zlib?expand=1

Although it looks like none of the non-pinned ones were updated.  This makes me think that it one of our pinned deps upgraded in place and stopped requiring zlib.  We were bringing it in automatically before.
